src/webxdc.rs

```diff
@@ -284,6 +278,22 @@ impl Context {
         Ok(status_update_serial)
     }
 
+    pub(crate) async fn internal_write_status_update(
```

Comment:
What does "internal" mean here?

Reply:
It means sth. like "inner", maybe we should rename it to "write_status_update_inner"